cli/config/credentials/file_store.go

```diff
@@ -74,7 +74,10 @@ func ConvertToHostname(maybeURL string) string {
 	if strings.Contains(stripped, "://") {
 		u, err := url.Parse(stripped)
 		if err == nil && u.Hostname() != "" {
-			return u.Hostname()
+			if u.Port() == "" {
+				return u.Hostname()
+			}
+			return u.Hostname() + ":" + u.Port()
```

Comment:
If we can expect IPv6 addresses, we should probably use `netJoinHostPort()` here;

```go
return net.JoinHostPort(u.Hostname(), u.Port())
```